Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -161,19 +170,7 @@ open class ChatChannelListVC: _ViewController,
         willDisplay cell: UICollectionViewCell,
         forItemAt indexPath: IndexPath
     ) {
-        if controller.state != .remoteDataFetched {
-            return
-        }
-
-        guard collectionView.isTrackingOrDecelerating else {
-            return
-        }
-
-        if indexPath.row < collectionView.numberOfItems(inSection: 0) - 10 {
```

Comment:
Not sure if the collectionView can have less than 10 items but still require to loadMore. 

If yes, then this check will cause issues